tests/qos/args/qos_sai_args.py

```diff
@@ -46,3 +46,10 @@ def add_qos_sai_args(parser):
         help="QoS SAI comma separated list of source ports. Test currently expects exactly 1 source port",
     )
 
+    qos_group.addoption(
+        "--qos_dual_tor",
+        action="store",
+        type=str2bool,
+        default=False,
+        help="Test QoS on dual ToR ports"
+    )
```

Comment:
Can you please clarify why do we need this option? Can we set `is_dual_tor = True` when `dualtor_ports` is not empty?
I have the ask because when we are running nightly tests with pipeline, we are normally using the same options for all testbeds.

Reply:
Sure.
We need to test both scenarios (with/without additional lossless PG/Q) even if `dualtor_ports` is not empty, right?

Reply:
Yes, that makes sense. Thanks